### ISSUE TITLE
Add 'newc' file format

### ISSUE THREAD

**Issue by ninjahacker** [radare/radare2#2332]
The 'newc' file format is used for initial boot filesystem images. It could be nice to have support for identifying these. Maybe it isn't very high demand though? The specs are pretty well labeled:

https://www.kernel.org/doc/Documentation/early-userspace/buffer-format.txt

I may add it if no one else does once I add more features to the compressed Linux kernel (zimg) format.

**Comment by jvoisin** [comment]
Initial Linux kernel (zimg) support added in [this]( https://github.com/radare/radare2/commit/d1351cf836df3e2e63043a6dc728e880316f00eb ) commit.

**Comment by ninjahacker** [comment]
Yes, that was me. I was just saying that making the zimg support better is a higher priority than this, so I likely won't be doing it soon.

**Comment by jvoisin** [comment]
I know, I added this for people reading/following this feature

**Comment by Maijin** [comment]
@jvoisin :baby_chick:

**Comment by ret2libc** [comment]
This issue has been moved from radareorg/radare2 to radareorg/ideas as we are trying to clean our backlog and this issue has probably been created a long while ago. This is an effort to help contributors understand what are the actionable items they can work on, prioritize issues better and help users find active/duplicated issues more easily. If this is not an enhancement/improvement/general idea but a bug, feel free to ask for re-transfer to main repo. Thanks for your understanding and contribution with this issue.
